main/src/com/google/refine/model/changes/RowAdditionChange.java

```diff
@@ -81,18 +81,22 @@ public void revert(Project project) {
     @Override
     public void save(Writer writer, Properties options) throws IOException {
 
+        writeFieldAndValue(writer);
+        for (Row row : _additionalRows) {
+            row.save(writer, options);
+            writer.write('\n');
+        }
+        writer.write(EOC + "\n");
+    }
+
+    private void writeFieldAndValue(Writer writer) throws IOException {
```

Comment:
Since this method isn't reused anywhere, I actually prefer a linear flow. If there were multiple copies of the code which could be extracted into a single method and replaced with calls to it, that would be a different story, but since it's only used once, it's on the small side to extract into a method.